package.json

```diff
@@ -76,11 +81,20 @@
     "prepublish": "npm run build",
     "test": "npm run lint && jest",
     "test:watch": "jest --watchAll",
+    "test:current": "jest --watch",
     "storybook": "start-storybook -c storybook -p 6006",
     "build-storybook": "build-storybook -c storybook -o .out",
     "storybook:deploy": "storybook-to-ghpages",
     "semantic-release": "semantic-release pre && npm publish && semantic-release post"
   },
+  "jest": {
+    "modulePathIgnorePatterns": [
+      "lib"
+    ],
+    "setupFiles": [
```

Comment:
Does this works with older versions of react?

Reply:
I misread the [docs](http://facebook.github.io/jest/docs/en/configuration.html#modulepathignorepatterns-array-string) originally. It should be `["<rootDir>/lib/"] instead.

It seems React `^15.6.2` and Jest `^19.0.2` can still accept `modulePathIgnorePatterns`. I did a simple test in this [branch](https://github.com/priley86/patternfly-react/tree/react-15-test) in my fork. I think it is only dev dependency though, so not sure it matters. I am only adding this to remove the warnings about duplicate manual mock when Jest runs currently in this repo...